app/models/pipeline_run.rb

```diff
@@ -723,7 +729,7 @@ def generate_aggregate_counts(tax_level_name)
     uncategorizable_id = TaxonLineage::MISSING_LINEAGE_ID.fetch(tax_level_name.to_sym, -9999)
     uncategorizable_name = "Uncategorizable as a #{tax_level_name}"
     TaxonCount.connection.execute(
-      "INSERT INTO taxon_counts(pipeline_run_id, tax_id, name,
+      "REPLACE INTO taxon_counts(pipeline_run_id, tax_id, name,
```

Comment:
do you want REPLACE INTO or INSERT IGNORE INTO? are these always the exact same rows or are they sometimes actually being updated?

Reply:
yes. i think so because we have NT and NT+ now.